docs/example-scenario/forensics/index.md

```diff
@@ -148,6 +170,9 @@ The Copy-VmDigitalEvidence runbook performs the following actions:
 
 ### Evidence retrieval
```

Comment:
As discussed please improve this section

Reply:
@elanzel as agreed please improve Evidence retrieval

If you would like to insert references to private link picture must be changed as well